edge-rtac/src/main/resources/domain/edge-rtac/rtac-junit.feature

```diff
@@ -0,0 +1,20 @@
+Feature: mod-circulation integration tests
+
+  Background:
+    * url baseUrl
+    * table modules
+      | name                      |
+      | 'mod-login'               |
+      | 'mod-permissions'         |
+      | 'edge-rtac'               |
+      | 'mod-inventory'           |
+      | 'mod-inventory-storage'   |
+
+    * table adminAdditionalPermissions
+      | name                      |
+
+    * table userPermissions
+      | name                      |
```

Comment:
These tests create instances, holdings, items, locations etc, how does it do that when there are no permissions granted to the users?

Reply:
with testAdmin there is no need to add any permissions, whatever modules are added in *** table modules** by default all permissions will get added automatically to the user. 
Let me know if thats okay or we should use **testUser** only.

Reply:
> with testAdmin there is no need to add any permissions, whatever modules are added in *** table modules** by default all permissions will get added automatically to the user.

Oh, I didn't know that was done automatically.

> Let me know if thats okay or we should use testUser only.

Given we have the separation. I would expect that all of the preparation e.g. creating items, fees / fines etc to be done by the administrative user and the use of the API being tested by the regular user.

Is that how these tests work at the moment?